src/collectors/kafkastat/kafkastat.py

```diff
@@ -128,7 +128,8 @@ def query_mbean(self, objectname, key_prefix=None):
                         key_prefix = key_prefix + '.' + key
 
         metrics = {}
-
+        trans_table = string.maketrans(', ','__')
```

Comment:
Seems this change broke travis tests:

https://travis-ci.org/python-diamond/Diamond/builds/108056178